nomic/dataset.py

```diff
@@ -753,6 +755,12 @@ def __init__(
         * **dataset_id** - An alternative way to load a dataset is by passing the dataset_id directly. This only works if a dataset exists.
         """
         assert identifier is not None or dataset_id is not None, "You must pass a dataset identifier"
+        # Normalize identifier.
+        if identifier is not None:
+            identifier = unicodedata.normalize("NFD", identifier)  # normalize accents
+            identifier = identifier.lower().replace(" ", "-").replace("_", "-")
+            identifier = re.sub(r"[^a-z0-9-]", "", identifier)
+            identifier = re.sub(r"-+", "-", identifier)
```

Comment:
We should probably replicate this functionality in ts-nomic too.

Reply:
Shouldn't be necessary: ts-nomic doesn't overload the slug and dataset identifiers into the same namespace like this. And atlas-next includes slug previews I think.